src/server.js

```diff
@@ -36,11 +36,17 @@ export default function start(
 	// (process.env.NODE_ENV === 'production')
 	require('source-map-support').install();
 
+	const pingRoute = {
+		path: '/ping',
+		method: 'GET',
+		handler: (request, reply) => reply('pong!'),
```

Comment:
just realized this will also need a `config: { auth: false }` property so that it doesn't try to generate an anonymous oauth token for every request :/